modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/QuantumToolkit.java

```diff
@@ -1324,6 +1325,25 @@ public void installInputMethodRequests(TKScene scene, InputMethodRequests reques
             this.image = image;
         }
 
+        QuantumImage(PixelBuffer<ByteBuffer> pixelBuffer) {
+            PixelFormat pf;
+            System.err.println("$$$$ pixelFormat = " + pixelBuffer.getPixelFormat());
```

Comment:
What about only printing this is quantum.verbose is true (using the verbose that is already on QuantumToolkit? )

Reply:
I think this is just a temporary debug statement, and probably not needed for the final patch.

Reply:
It is removed in the update.